__tests__/api.test.js

```diff
@@ -128,6 +128,29 @@ describe('GET /api/articles/:article_id', () => {
     })
 })
 
+describe('POST /api/articles/:article_id/comments', () => {
+    test('responds with a 201 status code and posted comment', () => {
+        const newComment = {
+            username: 'butter_bridge',
+            body: 'This is a new comment'
+        }
+        return request(app)
+        .post('/api/articles/1/comments')
+        .send(newComment)
+        .expect(201)
+        .then(res => {
+            expect(res.body.comment).toMatchObject({
+                comment_id: expect.any(Number),
+                body: 'This is a new comment',
+                article_id: 1,
+                author: 'butter_bridge',
+                votes: expect.any(Number),
+                created_at: expect.any(String)
+            })
+        })
+    })
+})
```

Comment:
One happy path test missing here: It's possible for our client to have extra properties on the post body
eg: {
            username: 'butter_bridge',
            body: 'This is a new comment',
            isKevCool: true
        }
status 201: ignore unnecessary properties